bin/secrets/init-credentials.sh

```diff
@@ -47,6 +47,15 @@ init_jenkins_credentials(){
     done
 }
 
+init_spi_credentials(){
+    vault read -field=value $TEST_SPI_CREDENTIALS_PATH > $basedir/spi.ini
+    for slave in vivid-1 xenial-1 xenial-2 xenial-3
+    do
+        docker cp $basedir/spi.ini compose_jenkins-slave-${slave}_1:/home/jenkins-slave/spi.ini
```

Comment:
With the latest version of docker this should be `jenkins_` instead of `compose_`